application/common/config/main.php

```diff
@@ -189,6 +191,24 @@
                     'prefix' => $logPrefix,
                     'exportInterval' => 1,
                 ],
+                [
+                    'class' => SentryTarget::class,
+                    'enabled' => true,
+                    'dsn' => Env::get('SENTRY_DSN'),
+                    'levels' => ['error'],
+                    'context' => true,
+                    // Additional options for `Sentry\init`
+                    // https://docs.sentry.io/platforms/php/configuration/options
+                    'clientOptions' => [
+                        'attach_stacktrace' => false, // stack trace identifies the logger call stack, not helpful
+                        'environment' => YII_ENV,
+                        'release' => 'idp-id-broker@6.6.0',
```

Comment:
If you want, we can have our CI/CD build process write a file with the current version number in it (from the git tag), and then you could read that file in order to dynamically show the current version number here, without the need to manually update this file with each release.

We do something similar in another project's Dockerfile, though this is to show the build date, not the version number:
```Dockerfile
# Record now as the build date/time (in a friendly format).
RUN date -u +"%B %-d, %Y, %-I:%M%P (%Z)" > /data/protected/data/version.txt
```

To get the git tag, I think we'd just need to add a build arg, to be supplied by Codeship at build time.

Just an idea.

Reply:
Yeah, I had that thought, but it seems like every time I try to do something like that it takes me numerous attempts to get it right. I was less interested in going through that this time.